lib/web_ui/lib/src/engine/canvaskit/renderer.dart

```diff
@@ -28,6 +28,13 @@ import 'shader.dart';
 import 'text.dart';
 import 'vertices.dart';
 
+/// Whether we can use client-provided ICU information instead of CanvasKit's
+/// built-in ICU.
+const bool useClientICU = false;
```

Comment:
I think I would name this something a little different, like `browserSupportsCanvaskitChromium` or something like that. This bool will only contain the information about whether *can* use the lite version of canvaskit, but the user might override that.

Reply:
Done